src/styles/partials/components/_page-header.scss

```diff
@@ -0,0 +1,46 @@
+.dg_page-header {
+	//background-image: url(/sebin/r/t/prototype-subpage-landing-header-bkg.jpg);
+	background-repeat: no-repeat;
+	background-size: cover;
+	background: $pale-blue;
+	color: $white;
+	padding: $padding-largest;
+	text-decoration: none;
+	p {
+		color: $white;
+		font-family: $default-body-font;
+		font-size: $h2-font-size;
+		font-style: italic;
+		padding-top: 25px;
+	}
+
+	.dg_page-header__box {
+		border-width: 10px;
```

Comment:
this is not doing anything, since you haven't specified a border color or style. You can remove.

Reply:
done